common/dtd/ldml.dtd

```diff
@@ -3170,6 +3170,8 @@ CLDR data files are interpreted according to the LDML specification (http://unic
     <!--@METADATA-->
 
 <!ELEMENT foreignSpaceReplacement ( #PCDATA ) >
+<!ATTLIST foreignSpaceReplacement xml:space (default | preserve) "preserve" >
+    <!--@VALUE-->
```

Comment:
`@VALUE` is probably not right here. THis should be handled globally because it is part of XML. It's actually not a 'value' of this xpath, its' structural.  CLDR-15671

Reply:
Agreed, but as I recall I was getting a unittest error without it. Also I think SurveyTool is not paying attention to this attribute and is not actually displaying the space.